src/qtt/algorithms/functions.py

```diff
@@ -340,24 +341,46 @@ def cost_function(params):
     return fitted_parameters
 
 
+def damped_sine_wave(x_data, amplitude, decay_rate, power, frequency, phase, offset):
+    """
+    A sine wave with exponential decay of a power of x.
+    Function:
+    $$ amplitude * exp(-(x_data*decay_rate)**power) * sin(2pi*frequency * x_data + phase) + offset $$
+    Function parameters:
+
+    Args:
+        x_data (np.ndarray): array of independent variable values where the function needs to be evaluated.
+        amplitude: amplitude of the sine at x=0.
+        decay_rate (Hz): value of 1/x where the damping factor is 1/e
+        power (-): power of x_data*decay_rate inside the exponent.
+        frequency (Hz)
```

Comment:
```suggestion
        frequency (float): Frequency in Hz
```
Similar remarks for the other parameters